bundles/org.openhab.binding.sonos/src/main/java/org/openhab/binding/sonos/internal/handler/ZonePlayerHandler.java

```diff
@@ -1468,6 +1477,51 @@ public boolean isOutputLevelFixed() {
         return stateMap.get("MusicSurroundLevel");
     }
 
+    public @Nullable String getCodec() {
+        String codec = stateMap.get("HTAudioIn");
+        switch (codec) {
+            case "0":
+                codec = "noSignal";
+                break;
+            case "21":
+                codec = "noSignal";
+                break;
+            case "22":
+                codec = "silence";
+                break;
+            case "32":
+                codec = "DTS";
+                break;
+            case "59":
+                codec = "dolbyAtmos";
+                break;
+            case "63":
+                codec = "dolbyAtmos";
+                break;
+            case "33554454":
+                codec = "silence20";
+                break;
```

Comment:
Why not just using `silence` ? We don't really care about the number of channels in this case.